lib/src/configuration.dart

```diff
@@ -744,7 +786,11 @@ class SyncResolveError extends SyncError {
   /// The numeric value indicating the type of the network resolution sync error.
   SyncResolveErrorCode get code => SyncResolveErrorCode.fromInt(codeValue);
 
-  SyncResolveError(String message, SyncErrorCategory category, SyncResolveErrorCode errorCode) : super(message, category, errorCode.index);
+  SyncResolveError(
```

Comment:
Should be deprecated

Reply:
The class `SyncResolveError` is already deprecated. I think we don't need to deprecate the constructor here.

Reply:
ok I might have missed that